etc/sample.openstack.yaml

```diff
@@ -5,108 +5,97 @@ site:
 
 compute:
     # Total number of cores available
-    total_cores: 0
+    #total_cores: 0
```

Comment:
It would be cleaner to have everywhere `# total _ram` instead of `#total_ram`. And I'm wondering/sceptic with all those commented lines, it's a bit confusing. But we certainly want to have things documented somewhere...

Reply:
Added spaces and reorganised the config file, first mandatory stuff, then optional